$MirOS: ports/www/firesomething/patches/patch-config_config_mk,v 1.5 2008/05/21 12:23:23 tg Exp $
--- config/config.mk.orig	2008-06-25 01:29:46.000000000 +0200
+++ config/config.mk	2008-08-24 14:50:13.000000000 +0200
@@ -131,7 +131,7 @@ endif
 # but save the version to allow multiple versions of the same base
 # platform to be built in the same tree.
 #
-ifneq (,$(filter FreeBSD HP-UX IRIX Linux NetBSD OpenBSD OSF1 SunOS,$(OS_ARCH)))
+ifneq (,$(filter FreeBSD HP-UX IRIX Linux MidnightBSD MirBSD NetBSD OSF1 OpenBSD SunOS,$(OS_ARCH)))
 OS_RELEASE	:= $(basename $(OS_RELEASE))
 
 # Allow the user to ignore the OS_VERSION, which is usually irrelevant.
